src/ralph/lib/permissions/models.py

```diff
@@ -19,24 +19,46 @@ def __new__(cls, name, bases, attrs):
             cls, name, bases, attrs
         )
         for field in new_class._meta.fields:
+            blacklist = new_class.Permissions.blacklist
             name = field.name
-            new_class._meta.permissions.append((
-                'can_edit_{}_field'.format(name),
-                _('Can edit {} field').format(name)
-            ))
+            if not field.primary_key and name not in blacklist:
+                new_class._meta.permissions.append((
+                    'can_edit_{}_{}_field'.format(
+                        new_class.__name__.lower(),
```

Comment:
is not necessary.. ``Permission`` models contains ``content_type`` (https://github.com/django/django/blob/1.8.1/django/contrib/auth/models.py#L65)